kv/dist_sender.go

```diff
@@ -481,12 +481,10 @@ func (ds *DistSender) Send(call client.Call) {
 	retryOpts := ds.rpcRetryOptions
```

Comment:
could you add to the comment of Send:

```
// This may temporarily adjust the request headers, so the client.Call
// must not be used concurrently until Send has returned.
```